docs/conf.py

```diff
@@ -98,8 +99,20 @@
 # html_theme_options = {}
 
 html_sidebars = {
-    'index': ['sidebarintro.html', 'sourcelink.html', 'localtoc.html', 'searchbox.html', 'hacks.html'],
-    '**': ['sidebarintro.html', 'sourcelink.html', 'globaltoc.html', 'searchbox.html', 'hacks.html']
+    'index': [
+        'sidebarintro.html',
+        'sourcelink.html',
+        'localtoc.html',
+        'searchbox.html',
+        'hacks.html',
+    ],
+    '**': [
+        'sidebarintro.html',
+        'sourcelink.html',
+        'globaltoc.html',
+        'searchbox.html',
+        'hacks.html',
+    ],
```

Comment:
If the list/tuple/set doesn't match max-line-length, then each parameter writes ones on a row.